### ISSUE TITLE
Azure Flexible Postgres

### ISSUE THREAD

**Issue by viddagrava** [cybertec-postgresql/pg_squeeze#74]
Hey,

This is rather a question, then a real issue.
I have an azure flexible postgres server, where the wal_level setting currently is replica.

Are there any specific consequences of changing that?
Secondly, could one use pg_squeeze with this current setting?

Best,
David

**Comment by kovmir** [comment]
> wal_level setting currently is replica. Are there any specific consequences of changing that?

Yes, [there are](https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-WAL-LEVEL). `replica` is the default, and is less verbose.

> Secondly, could one use pg_squeeze with this current setting?

No, one cannot.
